From 117bccdc908b571e1b5ef4fd7890398908b7ad9c Mon Sep 17 00:00:00 2001
From: "Lopez Esparza, Ricardo" <ricardo.lopez.esparza@intel.com>
Date: Fri, 7 Aug 2015 22:24:56 -0500
Subject: [PATCH] Changing acpi ID to the sensor ov2685

OV2685 driver uses INT33BE as acpi device ID, a value
already used in BYT for the sensor OV5693. In order to
avoid loading the incorrect driver, we need to change
this value to XXOV2685. The new value also follows the
pattern used by other OV sensors.

Change-Id: I7fde1513f95b88afcd1676c9bcbd7b125105e214
Original-Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-15801
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-7449
Signed-off-by: Lopez Esparza, Ricardo <ricardo.lopez.esparza@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/68467
Build: IMIN (SC)
Reviewed-by: Tuukka Toivonen <tuukka.toivonen@intel.com>
Approver: Yunliang Ding <yunliang.ding@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/ov2685.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov2685.c b/drivers/external_drivers/camera/drivers/media/i2c/ov2685.c
index 7e0ad0e..6ef25cb 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov2685.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov2685.c
@@ -1919,7 +1919,7 @@ out_free:
 MODULE_DEVICE_TABLE(i2c, ov2685_id);
 #ifdef CONFIG_GMIN_INTEL_MID
 static struct acpi_device_id ov2685_acpi_match[] = {
-	{ "INT33BE" },
+	{ "XXOV2685" },
 	{},
 };
 MODULE_DEVICE_TABLE(acpi, ov2685_acpi_match);
-- 
1.9.1

